clouddrift/select.py

```diff
@@ -1,40 +1,41 @@
-import awkward as ak
+import xarray as xr
 import numpy as np
+import warnings
 
 
-def mask_var(var: ak.Array, value):
+def mask_var(var: xr.DataArray, value):
     """Return the mask of a subset of the data matching a test criterion.
 
     Args:
-        var (ak.Array): ak.Array
+        var (xr.DataArray): ak.Array
         value: tuple, list or scalar defining a test criterion
 
     Returns:
-        mask (Ak.Array): where values corresponding to the criterion are True
+        mask (xr.DataArray): where values corresponding to the criterion are True
     """
 
     if isinstance(value, tuple):  # min/max defining range
         mask = np.logical_and(var >= min(value), var <= max(value))
     elif isinstance(value, list):  # select multiples
-        mask = ak.zeros_like(var)
+        mask = xr.zeros_like(var)
         for v in value:
             mask = np.logical_or(mask, var == v)
     else:  # select one
         mask = var == value
     return mask
 
 
-def subset(ds: ak.Array, criteria: dict) -> ak.Array:
+def subset(ds: xr.Dataset, criteria: dict) -> xr.Dataset:
     """Subset the dataset as a function of one or many criteria. The criteria are passed as a dictionary, where
```

Comment:
Docstring format;

Also we need to explain that this function expects `obs` and `traj` dimensions. That said, should we allow the user to specify some arbitrary dimension along which to subset? This would complicate the implementation though, so I'm not sure.